docs/model-dev-guide/api-guides/apis-howto/api-keras-ug.rst

```diff
@@ -7,129 +7,155 @@
 +=====================================================================+
 | :ref:`keras-reference`                                              |
 +---------------------------------------------------------------------+
 
-This document guides you through training a Keras model in Determined. You need to implement a trial
-class that inherits :class:`~determined.keras.TFKerasTrial` and specify it as the entrypoint in the
-:ref:`experiment-configuration`.
+This document guides you through training a Keras model in Determined.  You will need to update your
+``model.fit()`` call to include a :class:`~determined.keras.DeterminedCallback` and submit it to
+a Determined cluster.
 
-To learn about this API, you can start by reading the trial definitions in the `Iris categorization
-example
+To learn about this API, you can start by reading the ``train.py`` script in the `Iris
+categorization example
 <https://github.com/determined-ai/determined-examples/tree/main/computer_vision/iris_tf_keras>`__.
 
-***********
- Load Data
-***********
+**********************
+ Configure Entrypoint
+**********************
 
-.. note::
+Determined requires you to launch training jobs by submitting them with an
+:ref:`experiment-configuration` that tells the Determined master how to start your container.  For
+Keras training, you will always want to wrap your training script in Determined's :ref:`TensorFlow
+launcher <launch-tensorflow>`:
 
-   Before loading data, visit :ref:`load-model-data` to understand how to work with different
-   sources of data.
+.. code:: yaml
 
-Loading data is done by defining :meth:`~determined.keras.TFKerasTrial.build_training_data_loader`
-and :meth:`~determined.keras.TFKerasTrial.build_validation_data_loader` methods. Each should return
-one of the following data types:
+   entrypoint: >-
+     python3 -m determined.launch.tensorflow --
+     python3 my_train.py --my-arg...
 
-#. A tuple ``(x, y)`` of NumPy arrays. x must be a NumPy array (or array-like), a list of arrays (in
-   case the model has multiple inputs), or a dict mapping input names to the corresponding array, if
-   the model has named inputs. y should be a numpy array.
+Determined's TensorFlow launcher will automatically configure your training script with the right
+``TF_CONFIG`` environment variable for for distributed training when distributed resources are
+available, and will safely do nothing when they are not.
 
-#. A tuple ``(x, y, sample_weights)`` of NumPy arrays.
+****************************************************************
+ Obtain a ``det.core.Context`` and a ``tf.distribute.Strategy``
+****************************************************************
 
-#. A ``tf.data.dataset`` returning a tuple of either (inputs, targets) or (inputs, targets,
-   sample_weights).
+When using distributed training, TensorFlow requires you to create your ``Strategy`` early in the
+process lifetime, before creating your model.
```

Comment:
is there a reason why Strategy is capitalized? is it a proper noun?

Reply:
Yes, it is a symbol name in tensorflow.  That's why it is an inline code block as well.